src/test/java/us/warframestat/moddetection/api/AppTest.java

```diff
@@ -10,6 +11,7 @@
 import java.util.Objects;
 
 import static org.junit.jupiter.api.Assertions.*;
+import static us.warframestat.moddetection.api.API.LOGGER;
```

Comment:
Capitalisation of packages? Feel free to correct me...

Reply:
also: api.API?

Reply:
i think the idea is that .api is the last package, .API is the class, .LOGGER is an inner class, so it makes general sense